src/migrations/20200926182637-create-user.js

```diff
@@ -25,14 +25,15 @@ module.exports = {
       },
       teamId: {
         type: Sequelize.STRING,
-      },
-      isVerified: {
-        type: Sequelize.BOOLEAN,
```

Comment:
We have migration errors because `isVerified` and `verificationHash` were added to the old migration and another migration was added doing the same thing